lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -37,6 +41,17 @@ class BootstrapWindowsWinrm < Bootstrap
       banner "knife bootstrap windows winrm FQDN (options)"
 
       def run
+        if (Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key])))
+
+          if !negotiate_auth? && !(locate_config_value(:winrm_transport) == 'ssl')
+            ui.error("Validatorless bootstrap only supported with negotiate authentication protocol and ssl/plaintext transport")
```

Comment:
The issue with the validator is that this is current functionality we can't break -- we can't raise an exception when you do this. This is new, so we can make this painful.

That said, perhaps the exception is the wrong approach, maybe should just do a really scary warning to discourage this and make it very clear this should not be done except for testing purposes.